### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#160] [Esercizi - Bertini] [answered]
![Screenshot 2023-10-23 185127](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/d7c03a8a-9597-4de5-9273-457c781e5713)
![Screenshot 2023-10-30 133842](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/d2e6087b-72b9-4bb6-803f-40b3e7a71d48)

**Comment by daromacarlo** [comment]
![Immagine 2023-10-26 184902](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/73708efd-659c-4910-8354-fd8c1a739f22)

**Comment by CiottoloMaggico** [comment, marked as answer]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/3ff53187-aaf5-43b3-9026-0e13d3398272)

**Comment by IonutCicio** [comment]
@Elia-Belli Seppure i risultati siano corretti il modello che hai costruito dovrebbe essere sbagliato. Sono d'accordo sul fatto che $\Omega = \\{0_C, 0_E, 1_C, 1_E\\}$, il problema è che $\mathbb{P}(\\{0_C\\}) \neq 0.94$ e $\mathbb{P}(\\{1_C\\}) \neq 0.91$ (Altrimenti $\mathbb{P}(\Omega)$ sarebbe > 1).

Sarebbe più corretto scrivere

$$\mathbb{P}(\\{ 0_C \\} \mid \\{0_C, 0_E\\}) = \frac{\mathbb{P}(\\{ 0_C\\} \cap \\{0_C, 0_E\\} ) }{ \mathbb{P}( \\{0_C, 0_E\\})} = \frac{\mathbb{P}(\\{ 0_C\\}) }{ \mathbb{P}( \\{0_C, 0_E\\})} =  0.94$$

Che si interpreta come _la probabilità che 0 sia corretto quando viene trasmesso uno 0_, e lo stesso vale anche per 1

$$\mathbb{P}(\\{ 1_C \\} \mid \\{1_C, 1_E\\}) = \frac{\mathbb{P}(\\{ 1_C\\} \cap \\{1_C, 1_E\\} ) }{ \mathbb{P}( \\{1_C, 1_E\\})} = \frac{\mathbb{P}(\\{ 1_C\\}) }{ \mathbb{P}( \\{1_C, 1_E\\})} =  0.91$$

In cui sto considerando l'evento _"è stato trasmesso 0"_ $= \\{0_C, 0_E\\}$ e l'evento _"è stato trasmesso 1"_ $= \\{1_C, 1_E\\}$, per cui, quando calcolo il punto _1)_ dell'esercizio ho, come hai giustamente considerato te, $\Omega_1 = \\{1_C, 0_E\\}$, e, per la formula delle probabilità totali

$$
\mathbb{P}(\\{1_C, 0_E\\}) = \mathbb{P}(\\{1_C\\} \mid \\{1_C, 1_E\\}) \cdot \mathbb{P}(\\{1_C, 1_E\\}) + (1 - \mathbb{P}(\\{0_C\\} \mid \\{0_C, 0_E\\})) \cdot \mathbb{P}(\\{0_C, 0_E\\}) =
$$

$$
 = \frac{\mathbb{P}(\\{ 1_C\\}) }{ \mathbb{P}( \\{1_C, 1_E\\})} \cdot \mathbb{P}(\\{1_C, 1_E\\}) + (1 - \frac{\mathbb{P}(\\{ 0_C\\}) }{ \mathbb{P}( \\{0_C, 0_E\\})} ) \cdot \mathbb{P}(\\{0_C, 0_E\\}) =
$$

$$
= \mathbb{P}(\\{1_C\\}) + \mathbb{P}(\\{0_E\\}) = 0.91 \cdot 0.55 + (1 - 0.94) \cdot 0.45 \approx 0.5275 $$

Procedimento simile per il punto _2)_, e dovrebbe venire $\approx 0.4725$ 

Un altro problema della tua soluzione, che deriva da come hai costruito il modello, è che $\mathbb{P}(1 \mid 1_C) = 1$, quindi la probabilità che ho ricevuto 1 dato l'evento "è stato trasmesso 1 correttamente" è 1: se è stato trasmesso 1 correttamente, non posso aver ricevuto 0. Proviamo ad analizzarlo con la definizione:

$$
\mathbb{P}(1 \mid 1_C) \equiv \mathbb{P}(\\{1_C, 1_E\\} \mid \\{1_C\\}) = \frac {\mathbb{P}(\\{1_C, 1_E\\} \cap \\{1_C\\})} {\mathbb{P}(\\{1_C\\})} = \frac{\mathbb{P}(\\{1_C\\})}{\mathbb{P}(\\{1_C\\})} = 1 
$$

_3)_ Per determinare la probabilità che sia stato trasmesso 1 se si è ricevuto 1, e conoscendo $\mathbb{P}(\\{1_C\\} \mid \\{1_C, 1_E\\})$ si può utilizzare la formula di Bayes

$$
\mathbb{P}(\\{1_C, 1_E\\} \mid \\{1_C, 0_E\\}) = \frac {\mathbb{P}(\\{1_C, 1_E\\} \cap \\{1_C, 0_E\\})} {\mathbb{P}( \\{ 1_C, 0_E \\} )} = \frac {\mathbb{P}(\\{ 1_C \\})} {\mathbb{P} (\\{ 1_C, 0_E \\})} =
$$

$$
= \frac {\mathbb{P}(\\{ 1_C \\} | \\{ 1_C, 1_E\\} ) \cdot \mathbb{P}(\\{1_C, 1_E \\})} {\mathbb{P} (\\{ 1_C, 0_E \\})} = \frac{0.91 \cdot 0.55}{0.5275} \approx 0.9488
$$

_4)_ Determinare la probabilità che sia stato trasmesso 0 se si è ricevuto 0. Simile al punto precedente

$$
\mathbb{P}(\\{0_C, 0_E\\} \mid \\{0_C, 1_E\\}) = \frac {\mathbb{P}(\\{0_C, 0_E\\} \cap \\{0_C, 1_E\\})} {\mathbb{P}( \\{ 0_C, 1_E \\} )} = \frac {\mathbb{P}(\\{ 0_C \\})} {\mathbb{P} (\\{ 0_C, 1_E \\})} =
$$

$$
= \frac {\mathbb{P}(\\{ 0_C \\} | \\{ 0_C, 0_E\\} ) \cdot \mathbb{P}(\\{0_C, 0_E \\})} {\mathbb{P} (\\{ 0_C, 1_E \\})} = \frac{0.94 \cdot 0.45}{0.4725} \approx 0.8952
$$

_5)_ Calcolare la probabilità che si verifichi un errore di trasmissione

$$\mathbb{P}(\\{0_E, 1_E\\}) = \mathbb{P}(\\{0_E\\}) + \mathbb{P}(\\{1_E\\}) =$$

$$
= (1 - \mathbb{P}(\\{ 0_C \\} \mid \\{0_C, 0_E\\})) \cdot \mathbb{P}(\\{0_C, 0_E\\}) + (1 - \mathbb{P}(\\{ 1_C \\} \mid \\{1_C, 1_E\\})) \cdot \mathbb{P}(\\{1_C, 1_E\\}) =
$$

$$
= (1 - 0.94) \cdot 0.45 + (1 - 0.91) \cdot 0.55 \approx 0.0765
$$

Analogo a @CiottoloMaggico
